Cypress is not available for 32bit and only used for tests

diff -Nurp a/src/pybind/mgr/dashboard/frontend/package.json b/src/pybind/mgr/dashboard/frontend/package.json
--- a/src/pybind/mgr/dashboard/frontend/package.json	2022-02-26 15:22:09.715994872 +0000
+++ b/src/pybind/mgr/dashboard/frontend/package.json	2022-02-26 15:22:56.975770423 +0000
@@ -119,8 +119,6 @@
     "@types/simplebar": "5.1.1",
     "@types/swagger-ui": "3.52.0",
     "codelyzer": "6.0.2",
-    "cypress": "9.0.0",
-    "cypress-multi-reporters": "1.5.0",
     "html-linter": "1.1.1",
     "htmllint-cli": "0.0.7",
     "identity-obj-proxy": "3.0.0",
